python/ray/data/_internal/execution/streaming_executor_state.py

```diff
@@ -259,16 +259,16 @@ def refresh_progress_bar(self, resource_manager: ResourceManager) -> None:
     def summary_str(self, resource_manager: ResourceManager) -> str:
         queued = self.num_queued() + self.op.internal_queue_size()
         active = self.op.num_active_tasks()
-        desc = f"- {self.op.name}: {active} active, {queued} queued"
+        desc = f"- {self.op.name}. Tasks: {active} 🟢, {queued} 🟡"
```

Comment:
"N queued" actually means N blocks in the input buffer, not number of tasks. 
(the previous was already a bit confusing)

Reply:
also since we have a "Tasks: " section here. I'm wondering maybe we can also move the actor info after this. Instead of at the very end.

Reply:
maybe format it as "Tasks ..., Actors ..., Input blocks ..."

Reply:
also, and blocked sign can be part of "tasks"

Reply:
> maybe format it as "Tasks ..., Actors ..., Input blocks ..."

+1 something like this seems reasonable to me